tests/test_utils.py

```diff
@@ -363,5 +335,51 @@ def test_tombstone_pages(self, mock_list_v2):
                 self.assertNotIn(x, page_one)
     # TODO add test to enumerate list and ensure all bundles that should be present are there.
 
+
+class TestUtils(unittest.TestCase):
+    @testmode.standalone
+    def test_countdown(self):
+        for seconds_remaining in countdown(1):
+            pass
+
+    @testmode.standalone
+    def test_circular_generator(self):
+        lst = ['a', 'b', 'c']
+        letters = iter(circular_generator(lst))
+        self.assertEqual('a', next(letters))
+        self.assertEqual('b', next(letters))
+        self.assertEqual('c', next(letters))
+        self.assertEqual('a', next(letters))
+
+    @testmode.standalone
+    def test_multipart_parallel_upload(self):
+        data = os.urandom(7 * 1024 * 1024)
+        metadata = {'something': "foolish"}
+        part_size = 5 * 1024 * 1024
+        s3_client = Config.get_native_handle(Replica.aws)
+        bucket = os.environ['DSS_S3_BUCKET_TEST']
+        with self.subTest("copy multiple parts"):
+            with io.BytesIO(data) as fh:
+                multipart_parallel_upload(
+                    s3_client,
+                    bucket,
+                    "fake_key",
```

Comment:
please use a key name that can link us back to this test.

Reply:
I'm going to revert this change and address your comment in a different PR.